lib/label.rb

```diff
@@ -0,0 +1,16 @@
+class Label
+  attr_accessor :title, :color
+  attr_reader :id, :items
+
+  def initialize(title, color)
+    @id = Random.rand(1..10_000)
+    @title = title
+    @color = color
+    @items = []
+  end
+
+  def add_item(item)
```

Comment:
Hello @ahangarha 
I think it would be better to set the` item label` before adding it to the `items list`

Reply:
Thanks for the review. I think this makes an infinite loop.